core/src/main/scala/com/stripe/dagon/HCache.scala

```diff
@@ -6,16 +6,6 @@ package com.stripe.dagon
 sealed class HCache[K[_], V[_]] private (init: HMap[K, V]) {
   private var hmap: HMap[K, V] = init
 
-  /**
```

Comment:
we weren't using or testing them, so i removed them. i can readd them with tests if we expect folks to use these types themselves (i am not sure i expect other people to do that beyond e.g. `Memoize`).